src/core/Global.ttslua

```diff
@@ -46,87 +49,46 @@ TOKEN_DATA = {
   clue     = {image = "http://cloud-3.steamusercontent.com/ugc/1758068501357164917/1D06F1DC4D6888B6F57124BD2AFE20D0B0DA15A8/", scale = {0.15, 0.15, 0.15}}
 }
 
-IMAGE_TOKEN_MAP = {
-  ["https://i.imgur.com/nEmqjmj.png"] = "Elder Sign",
-  ["https://i.imgur.com/uIx8jbY.png"] = "+1",
-  ["https://i.imgur.com/btEtVfd.png"] = "0",
-  ["https://i.imgur.com/w3XbrCC.png"] = "-1",
-  ["https://i.imgur.com/bfTg2hb.png"] = "-2",
-  ["https://i.imgur.com/yfs8gHq.png"] = "-3",
-  ["https://i.imgur.com/qrgGQRD.png"] = "-4",
-  ["https://i.imgur.com/3Ym1IeG.png"] = "-5",
-  ["https://i.imgur.com/c9qdSzS.png"] = "-6",
-  ["https://i.imgur.com/4WRD42n.png"] = "-7",
-  ["https://i.imgur.com/9t3rPTQ.png"] = "-8",
-  ["https://i.imgur.com/stbBxtx.png"] = "Skull",
-  ["https://i.imgur.com/VzhJJaH.png"] = "Cultist",
-  ["https://i.imgur.com/1plY463.png"] = "Tablet",
-  ["https://i.imgur.com/ttnspKt.png"] = "Elder Thing",
-  ["https://i.imgur.com/lns4fhz.png"] = "Auto-fail",
-  ["http://cloud-3.steamusercontent.com/ugc/1655601092778627699/339FB716CB25CA6025C338F13AFDFD9AC6FA8356/"] = "Bless",
-  ["http://cloud-3.steamusercontent.com/ugc/1655601092778636039/2A25BD38E8C44701D80DD96BF0121DA21843672E/"] = "Curse",
-  ["http://cloud-3.steamusercontent.com/ugc/1858293462583104677/195F93C063A8881B805CE2FD4767A9718B27B6AE/"] = "Frost"
+ID_URL_MAP = {
+  ['blue']    = {name = "Elder Sign", url = 'https://i.imgur.com/nEmqjmj.png'},
+  ['p1']      = {name = "+1", url = 'https://i.imgur.com/uIx8jbY.png'},
+  ['0']       = {name = "0", url = 'https://i.imgur.com/btEtVfd.png'},
+  ['m1']      = {name = "-1", url = 'https://i.imgur.com/w3XbrCC.png'},
+  ['m2']      = {name = "-2", url = 'https://i.imgur.com/bfTg2hb.png'},
+  ['m3']      = {name = "-3", url = 'https://i.imgur.com/yfs8gHq.png'},
+  ['m4']      = {name = "-4", url = 'https://i.imgur.com/qrgGQRD.png'},
+  ['m5']      = {name = "-5", url = 'https://i.imgur.com/3Ym1IeG.png'},
+  ['m6']      = {name = "-6", url = 'https://i.imgur.com/c9qdSzS.png'},
+  ['m7']      = {name = "-7", url = 'https://i.imgur.com/4WRD42n.png'},
+  ['m8']      = {name = "-8", url = 'https://i.imgur.com/9t3rPTQ.png'},
+  ['skull']   = {name = "Skull", url = 'https://i.imgur.com/stbBxtx.png'},
+  ['cultist'] = {name = "Cultist", url = 'https://i.imgur.com/VzhJJaH.png'},
+  ['tablet']  = {name = "Tablet", url = 'https://i.imgur.com/1plY463.png'},
+  ['elder']   = {name = "Elder Thing", url = 'https://i.imgur.com/ttnspKt.png'},
+  ['red']     = {name = "Auto-fail", url = 'https://i.imgur.com/lns4fhz.png'},
+  ['bless']   = {name = "Bless", url = 'http://cloud-3.steamusercontent.com/ugc/1655601092778627699/339FB716CB25CA6025C338F13AFDFD9AC6FA8356/'},
+  ['curse']   = {name = "Curse", url = 'http://cloud-3.steamusercontent.com/ugc/1655601092778636039/2A25BD38E8C44701D80DD96BF0121DA21843672E/'},
+	['frost']   = {name = "Frost", url = 'http://cloud-3.steamusercontent.com/ugc/1858293462583104677/195F93C063A8881B805CE2FD4767A9718B27B6AE/'}
```

Comment:
restructured this table to avoid duplication, URLs were used in 3 different places